code/modules/mob/living/living_say.dm

```diff
@@ -142,6 +142,9 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 			say_dead(original_message)
 			return
 
+	if(HAS_TRAIT(src, TRAIT_SOFTSPOKEN))
```

Comment:
Is there a signal we can hook onto instead? I don't like the surface area of this